pylint/checkers/refactoring/refactoring_checker.py

```diff
@@ -1594,6 +1602,15 @@ def _check_use_list_or_dict_literal(self, node: nodes.Call) -> None:
                 elif inferred.qname() == "builtins.dict" and not node.keywords:
                     self.add_message("use-dict-literal", node=node)
 
+    def _check_use_dict_literal(self, node: nodes.Call) -> None:
+        """Check if dict is created by using the literal {}."""
+        if not isinstance(node.func, astroid.Name):
+            return
+        if node.func.name == "dict" and not node.args and node.keywords:
+            inferred = utils.safe_infer(node.func)
+            if inferred.qname() == "builtins.dict":
+                self.add_message("use-dict-literal-without-kwargs", node=node)
```

Comment:
```suggestion
                self.add_message("use-dict-literal-without-kwargs", args=(code_suggestion), node=node)
```
Content of ``code_suggestion`` not included in this suggestion but it migth contain an ``.as_string()`` somewhere 😄